iOS/MMIWG2S/ViewControllers/FacesARViewController.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  FacesARViewController.swift
+//  MMIWG2S
+//
+//  Created by Callum Johnston on 10/25/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+import UIKit
+import ARKit
+
+class FacesARViewController: UIViewController {
+    
+    private let faceNodeName = "faceNode"
+    private let faceViewer = ARFaceViewerUI()
+    
+    private var sceneView: ARSCNView = ARSCNView()
+    private var currentFaceAnchor: ARFaceAnchor?
+    
+    private let imageBlack = UIImage(named: "Face.scnassets/arkit_hand_black.png")
+    private let imageRed = UIImage(named: "Face.scnassets/arkit_hand_red.png")
```

Comment:
❓ These still have the mesh on them. Will you be able to turn that off or edit it out? Or do you need me to do that?

Reply:
I'd be happy to take it out. I added it in for debugging, but doesn't really seem necessary